packages/@okta/vuepress-site/docs/guides/pwd-optional-overview/main/index.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Password optional overview
+---
+
+## Overview
+
+Most hacking-related breaches today involve weak or stolen passwords and credential security theft. Passwords are a security vulnerability because they can be easily stolen and are prone to phishing attacks. They're also not convenient either. Users often need to remember multiple passwords, change them regularly, and either memorize or store them securely somewhere.
+
+Ditch the password for a passwordless sign-in experience. Give your users the ability to authenticate with a fingerprint, YubiKey, email, phone, or another authenticator. These authenticators can be more secure – it's harder to steal a fingerprint than a password – and more convenient since users no longer have to memorize and manage their passwords.
+
+<div class="three-quarters">
+
+![Screenshot showing a sign-in page with a sign-up link.](/img/pwd-optional/pwd-optional-overview-xy-diagram.png)
+
+</div>
+
+## Integration guides
+
+Implement passwordless authentication in your app using Okta's Embedded solutions and replace the password with more secure and convenient [authenticators](/docs/guides/authenticators-overview/main/). Use the following integration guides to step through a simple set of use cases and start your passwordless journey.
+
+### Embedded SDK
+
+* <StackSnippet snippet="signupwithemail" inline/>
+* <StackSnippet snippet="signinwithemail" inline/>
+* <StackSnippet snippet="changeemail" inline/>
+
+### Embedded Widget
+
+* <StackSnippet snippet="widgetsignin" inline/>
+
+Also, check out the <StackSnippet snippet="bestpractices" inline/>, to learn about best practices that will help you build a solid passwordless sign-in solution.
+
+## See also
+
+* [Okta Passwordless Authentication](https://www.okta.com/passwordless-authentication/)
+* [Let’s (finally) say goodbye to passwords](https://www.okta.com/go-passwordless/)
+* [Passwordless authentication deployment guide](https://help.okta.com/oie/en-us/Content/Topics/identity-engine/guides/pwa/pwa-main.htm)
```

Comment:
This needs to be an help.okta alias

Reply:
Okay. Read through the Alias docs but going to need to pick your brain tomorrow about how to use these.

Reply:
I commented out that link for now.